lib/cocoapods/sandbox/file_accessor.rb

```diff
@@ -168,7 +168,7 @@ def self.vendored_frameworks_headers_dir(framework)
       #
```

Comment:
See above in the title line: the file permissions changed from `0644` to `0775` for this file. Please revert this change.